### PR TITLE
#339 implemented activate method

### PR DESCRIPTION
**Pull request by ammaratef45**
PR for #339 
    - Implemented the activate method body
    - Added a test
    - Edited the template

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#382](https://github.com/timelesslounge/timelessis/issues/382) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#382](https://github.com/timelesslounge/timelessis/issues/382) is assigned to @vladarefiev[/z](https://www.0crat.com/u/vladarefiev), here is [why](https://www.0crat.com/footprint/CF9LG1YCA/481bb5eb-a6b3-4a12-889a-08cc0eb09a6b); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

**Comment by vladarefiev** [comment]
@ammaratef45 Hello, looks good to me. Just a few comments. Pls check.

File: timeless/auth/views.py [hunk 1]
```diff
@@ -1,7 +1,4 @@
 """Auth views module.
-@todo #232:30min Implement activate() methods. Update html templates when
- methods are implemented. Create tests for activate() in it_auth_test.py.
- Architect must document what activate() method must do.
```
[the remark this answers is not in the document]
Comment (ammaratef45) [reply timeless/auth/views.py]: @vladarefiev I asked the ARC in the ticket and he told me.

**Comment by ammaratef45** [comment]
@vladarefiev Thanks. Please take a look.

**Comment by vladarefiev** [comment]
@ammaratef45
Thanks

**Comment by vladarefiev** [comment]
@rultor merge

**Comment by emilianodellacasa** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job was finished in 22 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @vladarefiev[/z](https://www.0crat.com/u/vladarefiev), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#382](https://github.com/timelesslounge/timelessis/issues/382) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa)

**Comment by ypshenychka** [comment]
@vladarefiev According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>Messages in a job [always](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html#4-avoid-noiseaddress-your-comments) start with a name of a user they are addressed to.

Please correct your [code review comment](https://github.com/timelesslounge/timelessis/pull/382#discussion_r259573655) by indicating an addressee in the beginning.

**Comment by vladarefiev** [comment]
@ypshenychka fixed

**Comment by ypshenychka** [comment]
@vladarefiev thanks

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +20 point(s) just awarded to @vladarefiev[/z](https://www.0crat.com/u/vladarefiev)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**